techlibs/xilinx/xc3sda_brams.txt

```diff
@@ -0,0 +1,33 @@
+# Spartan 3A DSP block RAM rules.
+
+bram $__XILINX_RAMB16BWER_TDP
```

Comment:
Is it more accurate to say that this is an `SDP` rule since Yosys doesn't currently support `TDP`?
I know this does not deviate from the norm and that this is necessary `xc6s_brams_map`, but do we need to update all the rules to be more accurate in this respect (doesn't mean it should be done here)

Reply:
It is a SDP rule like all our rules are, but it's called TDP because it passes "TDP" as the RAM_MODE for the underlying primitive (which does not support any modes *other* than TDP, but the 8-kbit version does, and we use that in both variants).